dev/tools/anomaly-traces-parser.el

```diff
@@ -0,0 +1,31 @@
+;; This Elisp snippet adds a regexp parser for the format of Anomaly
+;; backtraces (coqc -bt ...), to the error parser of the Compilation
+;; mode (C-c C-c: "Compile command: ..."). Once the
+;; coq-change-error-alist-for-backtraces function has run, file
+;; locations in traces are recognized and can be jumped from easily
+;; from the *compilation* buffer.
+
+;; You can just copy everything below to your .emacs and this will be
+;; enabled from any compilation command launched from an OCaml file.
+
+(defun coq-change-error-alist-for-backtraces ()
+  "Hook to change the compilation-error-regexp-alist variable, to
+   search the coq backtraces for error locations"
+  (interactive)
+  (progn
+    (setq compilation-error-regexp-alist-alist
+          (append
+           '((coq-backtrace
```

Comment:
Consider using `add-to-list` instead of `setq` + `append`; this will prevent duplicates form appearing in the list.